### PR TITLE
chore: update concrete-python to 2.2.0

### PR DESCRIPTION
**Pull request by fd0r**

### CODE REVIEW

**Comment by fd0r** [comment]
3rd time I get the same flaky, what do you think we should do here @RomanBredehoft ?

File: tests/torch/test_hybrid_converter.py [hunk 1]
```diff
@@ -53,12 +53,13 @@ def run_hybrid_model_test(
 
     # Compute average of these counts (the accuracy)
     accuracy = is_in.float().mean()
+    del accuracy
```
Comment (RomanBredehoft) [inline tests/torch/test_hybrid_converter.py:56]: why this del ? 😅

Comment (fd0r) [reply tests/torch/test_hybrid_converter.py:56]: Just to avoid my linter from saying that the variable was unused.

Comment (RomanBredehoft) [reply tests/torch/test_hybrid_converter.py:56]: ah yes ok got it

Comment (fd0r) [reply tests/torch/test_hybrid_converter.py:56]: actually not enough to have pcc happy

**Review by RomanBredehoft — APPROVED** [review]
thanks !

**Pull request merged**